### DISCUSSION TITLE
Portuguese (Portugal) language

### DISCUSSION THREAD

**Discussion by mansil** [mdn/mdn-community#466] [Localization] [answered]
HI, 

Would be possible to activate the **_Portuguese (Portugal) language_** please?

I was away because of some health problems but I am OK now and I would like to carry on with the work that I as doing within the Portuguese language.

Thank you all.

Kind regards,

Manuela Silva
@mansil

**Comment by bsmth** [comment, marked as answer]
Hi @mansil, sorry for the delay in replying. Firstly, I'm glad your health is better! 🎉 

I don't think un-freezing retired locales is very likely unless there's exceptional demand and you have enough support. As far as I'm aware, there must be a team. Would you be the only person working on `pt-PT` or do you have support?

Have you been in touch with any of the other @mdn/localization-team-leads or @mdn/yari-content-pt-br.

I did some digging and the locale was [retired / frozen here](https://github.com/mdn/translated-content/pull/43). There's more info here: https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#translated-content-peer-guidelines

Thanks a lot and welcome back! 😄

**Comment by pransh15** [comment]
Closing this as answered. ✅
